vaadin-side-nav-flow-parent/vaadin-side-nav-flow/src/main/java/com/vaadin/flow/component/sidenav/SideNavItem.java

```diff
@@ -393,6 +393,28 @@ public void setTarget(String target) {
         }
     }
 
+    /**
+     * @return true if this item should be ignored by the Vaadin router and
+     *         behave like a regular anchor.
+     */
+    public boolean isRouterIgnore() {
+        return getElement().getProperty("routerIgnore", false);
+    }
+
+    /**
+     * The routing mechanism in Vaadin by default intercepts all side nav items
+     * with relative URL. This method can be used make the router ignore this
+     * item and this way make it behave like a regular anchor and cause a full
```

Comment:
Nit: We can maybe split this sentence into two for clarity.

Reply:
Done